pkg/operator/vspherecontroller/driver_starter.go

```diff
@@ -115,6 +117,7 @@ func (c *VSphereController) createCSIDriver() {
 			trustedCAConfigMap,
 			c.apiClients.ConfigMapInformer,
 		),
+		WithConfigMapDaemonSetAnnotationHook("vsphere-csi-config", defaultNamespace, c.apiClients.ConfigMapInformer),
```

Comment:
Even though Daemonset spec does not have to be modified, it appears that drivers must be restarted for new `CSINode` values to be generated with updated topology values.